install_config/cluster_metrics.adoc

```diff
@@ -598,7 +598,7 @@ clusterInfo:
 ====
 
 Once you have updated and saved the webconsole-config configmap file, you must
-restart your {product-title} instance.
+restart your web console pods.
```

Comment:
Actually, web console pods will restart automatically to get the change after you have updated and saved the webconsole-config configmap file. So user need not to restart pods manually.

Reply:
You're right. The web console Deployment have the livenessProbe to monitor the webconsole-config configmap changes [here](https://github.com/openshift/openshift-ansible/blob/94e1abb1b422ca1c44a898cdb102d6c794561d44/roles/openshift_web_console/files/console-template.yaml#L73-L84). If the configmap changed, then the livenessProbe will failed and as a result the container will restart.
```yaml
          livenessProbe:
            exec:
              command:
                - /bin/sh
                - -c
                - |-
                  if [[ ! -f /tmp/webconsole-config.hash ]]; then \
                    md5sum /var/webconsole-config/webconsole-config.yaml > /tmp/webconsole-config.hash; \
                  elif [[ $(md5sum /var/webconsole-config/webconsole-config.yaml) != $(cat /tmp/webconsole-config.hash) ]]; then \
                    echo 'webconsole-config.yaml has changed.'; \
                    exit 1; \
                  fi && NSS_SDB_USE_CACHE=no curl -k -f https://0.0.0.0:8443/console/
```

Reply:
Until detecting the changes, at least it takes 30 seconds by the default config values of the liveness probes. 
```      failureThreshold: 3
periodSeconds: 10
```
We may replace "you must restart your {product-title} instance" with ""your web console pods will restart soon".
How about your thought @kalexand-rh ?

Reply:
If the default liveness probe configuration is unlikely to change, I suggest replacing lines 600 and 601 with something like this:

> After you update and save the *webconsole-config* configmap file, your web console pods restart after the delay set by the liveness probe configuration, which is 30 seconds by default.

If not, I might say something like this:

> After you update and save the *webconsole-config* configmap file, your web console pods restart after a short delay, which is usually less than a minute.

Reply:
@kalexand-rh The liveness probe can change, but it's unusual. And playbooks can overwrite it next time. So I think your first suggestion is better choice. Thanks a lot. :D

Reply:
@yanpzhan Could you PTAL again ?

Reply:
The update is much clearer now.

Reply:
@yanpzhan Thank you for your review !